src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -387,7 +387,8 @@ Scalar<DataVector> spin_function(
  * `unit_normal_vector` can
  * be found by raising the index of the one-form returned by
  * `StrahlkorperGr::unit_normal_one_form`.
- * The argument `ylm` is the `YlmSpherepack` of the `Strahlkorper`.
+ * The argument `strahlkorper` is used to call the `YlmSpherepack`
```

Comment:
Change to "The argument `strahlkorper` is the surface on which the spin magnitude is computed."